moveit_experimental/jog_arm/config/ur_simulated_config.yaml

```diff
@@ -0,0 +1,55 @@
+###############################################
+# Modify all parameters related to jogging here
+###############################################
+use_gazebo: true # Whether the robot is started in a Gazebo simulation environment
+
+## Properties of incoming commands
+command_frame:  world  # TF frame that incoming cmds are given in
+command_in_type: "speed_units" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
+scale: # Only used if command_in_type=="unitless"
+  linear:  0.2  # Max linear velocity. Meters per publish_period. Units is [m/s]
+  rotational:  0.1 # Max angular velocity. Rads per publish_period. Units is [rad/s]
+  joint: 0.1  # Max joint angular/linear velocity. Rads or Meters per publish period. Units is [rad/s] or [m/s].
+low_pass_filter_coeff: 2.  # Larger --> trust the filtered data more, trust the measurements less.
```

Comment:
I sorted the filter parameter here, because I assumed that it is related to the incoming command. 
Correct me, if this assumption is wrong.